pages/FilterAttribute/components/SingleSelect/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
```

Comment:
For single select i get this space at the top
![image](https://user-images.githubusercontent.com/5811286/41401655-b235f508-6fc0-11e8-8110-1d6efb6e1444.png)